### ISSUE TITLE
Create a Guide for Database Schema

### ISSUE THREAD

**Issue by atapas** [reactplay/docs#14]
@Sachin-chaurasiya Add details

**Comment by supminn** [comment]
Hi @Sachin-chaurasiya @atapas 
I would like to take this up. Could you assign it to me?

**Comment by ArslanYM** [comment]
Can I work on this?

**Comment by Sachin-chaurasiya** [comment]
> Can I work on this?

Assigned to you.

**Comment by atapas** [comment]
@ArslanYM based on our discussion last time, what's the next step on it?

**Comment by Abhishek-90** [comment]
Is this issue still on and available?

**Comment by supminn** [comment]
> Is this issue still on and available?

Yes, it's still open. Would you like to take it up?

**Comment by Abhishek-90** [comment]
> > Is this issue still on and available?
> 
> Yes, it's still open. Would you like to take it up?

Yes, I would like to take it up. But will require some assistance initially to understand the schema and relationships myself.

**Comment by priyankarpal** [comment]
can you update here? @ArslanYM

**Comment by shivasharma14** [comment]
Is this still open?

**Comment by mdnm18** [comment]
Hello @atapas @priyankarpal , I'm interested in documenting the database schema. I explored the project's source code to understand the models, fields, and relationships. I will create a clear and comprehensive guide based on my findings. Please consider assigning this to me.
